src/Provider/ApiServiceProvider.php

```diff
@@ -65,7 +65,7 @@ public function register()
             'Dingo\Api\Console\Command\Docs',
         ]);
 
-        if (class_exists('Illuminate\Foundation\Application', false)) {
+        if (class_exists('Illuminate\Contracts\Foundation\Application', false)) {
```

Comment:
> I'm not sure I follow you. laravel/lumen would load illuminate/contracts ? False.

https://github.com/laravel/lumen-framework/blob/5.2/composer.json#L19-L38

illuminate/auth requires illuminate/contracts
illuminate/config requires illuminate/contracts
illuminate/container requires illuminate/contracts

How is it `false`? You can check `composer show -i` on a Lumen installation and see `illuminate/contracts` is install (or look inside your `vendor` folder.

> You're only helping detract from the push this pull request needs to help myself and others get this package implemented!

I'm helping not to introduce a **buggy** PR to be merged in.